drivers/volume/portworx/portworx.go

```diff
@@ -718,6 +718,19 @@ func (d *portworx) CreateVolume(volName string, size uint64, haLevel int64) (str
 	return resp.VolumeId, nil
 }
 
+func (d *portworx) CreateVolumeUsingRequest(request *api.SdkVolumeCreateRequest) (string, error) {
+	volDriver := d.getVolDriver()
+	resp, err := volDriver.Create(d.getContext(), request)
+	if err != nil {
+		err = fmt.Errorf("error while creating volume because of: %v", err)
+		return "", err
```

Comment:
Just do `return  "", fmt.Errorf("error while creating volume because of: %v", err)` instead

Reply:
fixed